messagebus/src/main/java/com/yahoo/messagebus/ErrorCode.java

```diff
@@ -122,4 +122,14 @@ public static String getName(int error) {
         default                     : return "UNKNOWN(" + error + ")";
         }
     }
+    public static boolean isFatal(int code) {
+        return code >= FATAL_ERROR;
+    }
+    public static boolean isTransient(int code) {
+        return code >= TRANSIENT_ERROR;
```

Comment:
Should this be `code >= TRANSIENT_ERROR && code < FATAL_ERROR`?

Reply:
Good point. Fixed and added tests for correct categorization